etc/plf/standalone-customize.conf

```diff
@@ -60,6 +60,12 @@ if ${DEPLOYMENT_CHAT_ENABLED}; then
     JAVA_OPTS="${JAVA_OPTS} -Dchat.dbName=${EXO_CHAT_MONGODB_NAME}"
     JAVA_OPTS="${JAVA_OPTS} -Dchat.chatPassPhrase=${EXO_CHAT_MONGODB_NAME}"
 fi
+# eXo Onlyoffice addon
+if ${DEPLOYMENT_ONLYOFFICE_DOCUMENTSERVER_ENABLED}; then
+        CATALINA_OPTS="${CATALINA_OPTS} -Donlyoffice.documentserver.host=localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}"
```

Comment:
bad indentation for te 3 lines in the if statement

Reply:
fixed